tests/target-browser/target_browser_suite_test.go

```diff
@@ -0,0 +1,16 @@
+package target_browser_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/onsi/ginkgo/reporters"
+)
+
+func TestTargetBrowser(t *testing.T) {
+	RegisterFailHandler(Fail)
+	junitReporter := reporters.NewJUnitReporter("junit.xml")
```

Comment:
Name should have `target-browser-junit.xml` so that no two tests have same xml file. This creates issue over prow UI

Reply:
right, added this change. One correction though, it'll not go on prow, it'll be visible on github-action CI page.